src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -169,6 +170,14 @@ void scrollBreadth(double deltaBreadth) {
         setBreadthOffset(breadthOffset.get() + deltaBreadth);
     }
 
+    void scrollLengthToPixel(double pixel) {
+        setLengthOffset(pixel);
+    }
+
+    void scrollBreadthToPixel(double pixel) {
+        setBreadthOffset(pixel);
+    }
```

Comment:
When I studied how you handled scrolling when a user scrolled, it would eventually call ````setBreadthOffset(breadthOffset.get() + deltaBreadeth)````. Since the above method, as far as I understood, handled the scrolling but is private, I created those two public methods "scroll___ toPixel"